moc.tex

```diff
@@ -674,19 +674,16 @@ \subsubsection{ASCII serialization}
     1/1 2 4 2/12-14 21 23 25 8/
 \end{lstlisting}
 
-\begin{table}[!htbp]
 \begin{center}
-   {\small
-   \begin{tabular} { l l l l l l l l l }
+  {\small
+  \begin{tabular} { l l l l l l l l l }
    & 1/1   & 2  & 4  & 2/12-14 & 21 & 23 &  25 & 8/ \\
    & $\downarrow$ & $\downarrow$ & $\downarrow$ & $\downarrow$ & $\downarrow$ & $\downarrow$ & $\downarrow$ & $\downarrow$ \\
 Order:     & 1    & 1 & 1 & 2        & 2  &  2 & 2  & 8 \\
 Cell:      & 1    & 2 & 4 & 12 to 14 & 21 & 23 & 25 & \\
-\end{tabular}
-}
+\end{tabular}}
+MOC ASCII encoding

```

Comment:
I noticed that in the PDF version, `MOC ASCII encoding` appears to the right of the tabular info instead of underneath.  I don't see this as a big deal, but thought it would be nicer underneath.

I randomly found that adding a blank line above `MOC ASCII encoding` has the effect of moving the caption under the table in my pdf build.  That seems kludgy but might be OK here:
```suggestion

MOC ASCII encoding
```